src/rules/add-claim-based-on-relationships.js

```diff
@@ -0,0 +1,121 @@
+/**
+* @title Adds a claim based on relationships
+* @overview Adds a claim based on the relationships the subject has in Scaled Access
+* @gallery true
+* @category marketplace
+*
+* This rule adds a claim to the access token based on relationships the subject has in Scaled Access.
+* 
+* Example of a resulting claim:
+```
```

Comment:
Looks like we're missing an asterisk here.

Reply:
Yes, I've fixed that